apps/docs/src/.vitepress/components/ComponentCard.vue

```diff
@@ -0,0 +1,52 @@
+<script lang="ts" setup>
+export type ComponentCardProps = {
+  /** Component name. */
+  name: string;
+  /** If true an "Implemented" badge will be shown, "Planned" otherwise. */
+  implemented?: boolean;
+};
+
+const props = defineProps<ComponentCardProps>();
+</script>
+
+<template>
+  <article class="card">
+    <Badge
+      v-if="props.implemented"
+      class="card__badge card__badge--implemented"
+      text="Implemented"
+    />
+    <Badge v-else class="card__badge" text="Planned" type="info" />
+    <h4 class="card__title">{{ props.name }}</h4>
+  </article>
+</template>
+
+<style lang="scss" scoped>
+.card {
+  display: block;
```

Comment:
if you want to use block instead of flex, you don't have to define it as it is the default :)